### PR TITLE
Add missing bs58 dependency

### PR DESCRIPTION
**Pull request by PaulLeCam**
I got this issue trying to use the lib, and the unrelated change is to fix a compilation error I got with TS.

### CODE REVIEW

File: src/permissions.ts [hunk 1]
```diff
@@ -64,9 +64,11 @@ export default class Permissions {
...
     if (origin === SELF_ORIGIN) return true
     const currentPaths = this.get(origin)
-    return paths.reduce((acc: boolean, path: string) => {
-      return acc && currentPaths.includes(path)
-    }, Boolean(currentPaths))
+    return currentPaths == null
+      ? false
+      : paths.reduce((acc: boolean, path: string) => {
+          return acc && currentPaths.includes(path)
+        }, true)
```
Comment (oed) [inline src/permissions.ts:71]: Not sure I follow why this is needed? If `Boolean(currentPaths)` doesn't work, can't we just do `Boolean(currentPaths == null)`?

**Review by oed — COMMENTED** [review]
Thanks for adding the missing dep!

File: src/permissions.ts (hunk 1 above)
Comment (PaulLeCam) [reply src/permissions.ts:71]: It works at runtime but at compile time TS was complaining about `currentPaths.includes` saying `currentPaths` could be `null`. In practice it's not true but the TS compiler is not smart enough to infer this.

Comment (oed) [reply src/permissions.ts:71]: Can't we just do `currentPaths?.includes`?

Comment (PaulLeCam) [reply src/permissions.ts:71]: Yes we could do `!!currentPaths?.includes(path)` as `currentPaths?.includes` could return `null`, but I found it more readable this way. I can change it to `!!currentPaths?.includes(path)` though if you prefer?

Comment (oed) [reply src/permissions.ts:71]: yeah, I think that's clearer than the turnary. (or `Boolean(currentPaths?.includes(path))`?)

**Review by oed — APPROVED** [review]

**Pull request merged**